### PR TITLE
hotfix(condo): refactored useAllObjects hook

### PR DESCRIPTION
**Pull request by ekabardinsky**

### CODE REVIEW

**Review by Alllex202 — APPROVED** [review]

**Review by toplenboren — APPROVED** [review]

File: packages/codegen/generate.hooks.ts [hunk 1]
```diff
@@ -380,64 +380,58 @@ export function generateReactHooks<
...
-                const { data: fetchedData } = await fetchMore({
+                await fetchMore({
                     variables: {
                         skip: skip,
                     },
+                    updateQuery (previousData, { fetchMoreResult }) {
+                        // @ts-ignore
+                        const updatedObjs = [...previousData.objs, ...fetchMoreResult.objs]
+                        // @ts-ignore
+                        return { ...previousData, objs: updatedObjs, count: fetchMoreResult.count }
```
Comment (toplenboren) [inline packages/codegen/generate.hooks.ts:409]: why tsignore is here? Best to fix it

File: packages/codegen/generate.hooks.ts [hunk 2]
```diff
@@ -380,64 +380,58 @@ export function generateReactHooks<
...
-        }, [count, data.length, error, fetchMoreError, firstPageLoaded, loadMore, skip])
+            if (objs.length >= count) {
+                return
+            }
+
+            loadMore(objs.length)
+        }, [loadMore, objs.length, count])
 
-        const allDataLoaded = data.length === 0 ? objs.length === count : data.length === count
+        const allDataLoaded = objs.length >= count && !loading
```
Comment (toplenboren) [inline packages/codegen/generate.hooks.ts:429]: boolean stuff is best prefixed with is, but this is just a nitpick :-)

File: packages/codegen/generate.hooks.ts (hunk 1 above)
Comment (Alllex202) [reply packages/codegen/generate.hooks.ts:409]: The problem with types.
I'm not sure we'll do anything about it

File: packages/codegen/generate.hooks.ts (hunk 2 above)
Comment (Alllex202) [reply packages/codegen/generate.hooks.ts:429]: Yes, but this is an old name that is used everywhere, so we will not change it.

**Pull request merged**